modos/cli.py

```diff
@@ -45,6 +45,14 @@ class RdfFormat(str, Enum):
 
 cli = typer.Typer(add_completion=False)
 
+OBJECT_PATH_ARG = Annotated[
+    Path,
```

Comment:
__suggestion(blocking)__:
```suggestion
    str,
```
Parsing as `PosixPath` is going to break `modos create --endpoint http://localhost create s3://modos-demo/test`, because it converts the object_path to `s3:/modos-demo/test`. I think parsing as string would work, but maybe you have a better idea to parse both url and path?

Reply:
I agree, it makes sense to use strings in the CLI and do the validation only in MODO